packages/terra-clinical-label-value-view/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
 ## Unreleased
+* Changed
+  * Created a new attribute on the LabelValueView component - childOfDescriptionList
+  * Updated the LabelValueView to be 
+  *     - a React.Fragment with a term and a definition when the LabelValueView is already inside a description list (when childOfDescriptionList is sent with a "true" value)
+  *     - a description list when childOfDescriptionList is not sent (undefined) or sent as "false"
+  * Changed the component examples.
```

Comment:
```suggestion

* Changed
   * Created a new prop `childOfDescriptionList` on the LabelValueView component.
   * Updated the LabelValueView to be a React.Fragment with a term and a definition when the LabelValueView is already inside a description list (when childOfDescriptionList is sent with a "true" value)
   *     - a description list when childOfDescriptionList is not sent (undefined) or sent as "false"
   * Updated the component examples for new prop `childOfDescriptionList`.
```

Reply:
I made the change. Thank you for the suggestion!